changelog: dev plans update (#773)
CleverCloud/documentation: content/changelog/2025/09-01-backups-dev-db.md

---
title: Update on free databases plans
description: Starting October 1st, 2025, backups will only be available for paid databases plans
date: 2025-09-01
tags:
  - addons
  - backups
authors:
  - name: David Legrand
    link: https://github.com/davlgd
    image: https://github.com/davlgd.png?size=40
excludeSearch: true
---

> We provide free DEV plans for some of our managed databases (MySQL, PostgreSQL) and will continue to do so to help you discover our platform. However, we'll only provide backups for paid plans starting October 1st, 2025. 

Reviewer: ```suggestion
We currently provide free DEV plans only for our MySQL and PostgreSQL managed databases and will continue to do so to help you discover our platform. However, we'll only provide backups for paid plans starting October 1st, 2025. 
```

> If you need 7-days included backups and production-ready service, consider upgrading from a DEV to [XXS Small Space or higher plan](https://www.clever-cloud.com/pricing/).

Reviewer: ```suggestion
If you need a 7-days included backups and production-ready service, consider upgrading from a DEV to [XXS Small Space or an higher plan](https://www.clever-cloud.com/pricing/).
```
